app/components/project-geometries/types/commercial-overlays.js

```diff
@@ -145,6 +145,7 @@ export default class CommercialOverlayComponent extends BaseClass {
 
   // this is wrong because it doesn't honor the correct target
   // it should be using the model's API, not passing stuff in directly
+  // this should also probably just happen in the API, when it gets posted!
```

Comment:
did you capture these things that should happen server-side in an issue?

Reply:
@julialucyhogan sure did! https://github.com/NYCPlanning/labs-applicantmaps/issues/463